implementation/src/main/java/io/smallrye/jwt/build/JwtEncryption.java

```diff
@@ -54,8 +54,8 @@
      * A key of size 2048 bits or larger MUST be used with the 'RSA-OAEP' and 'RSA-OAEP-256' algorithms.
      *
      * @return signed JWT token
```

Comment:
Hi @bobbyphilip Can you also please replace `signed' with `encrypted` and `signing` (operation) with `encryption`